src/ctia/init.clj

```diff
@@ -88,7 +87,6 @@
   "Lower-level Trapperkeeper booting function for
   custom services and config."
   [{:keys [services config]}]
-  (validate-entities)
```

Comment:
Moved to schema check on the return value of `ctia.entity.entities/all-entities`.